fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -168,7 +170,7 @@ class ApplyMetadataMutationsImpl {
 		} else {
 			ASSERT(cipherKeys != nullptr);
 			Arena arena;
-			toCommit->writeTypedMessage(m.encryptMetadata(*cipherKeys, arena));
+			toCommit->writeTypedMessage(m.encryptMetadata(*cipherKeys, arena, encryptionTime));
```

Comment:
nit: `encryptionTime` -> `encryptionCpuTime`, given we are separately accounting for EncryptionKey lookups? Does it make sense? If yes, then may be change at other places as well.